scripts/eval/eval.py

```diff
@@ -160,123 +159,75 @@ def evaluate_model(
         torch.cuda.synchronize()
     b = time.time()
 
-    log.info(f'Ran {model_cfg.model_name} eval in: {b-a} seconds')
+    log.info(f'Ran {model_name} eval in: {b-a} seconds')
     return (trainer, logger_keys, eval_gauntlet_callback, eval_gauntlet_df)
 
 
 def main(cfg: DictConfig) -> Tuple[List[Trainer], pd.DataFrame]:
+    cfgs: Tuple[Dict[str, Any], EvalConfig] = make_dataclass_and_log_config(
+        cfg, EvalConfig, EVAL_CONFIG_KEYS, icl_tasks_required=True)
+    logged_cfg, eval_config = cfgs
```

Comment:
the tuple unpacking just be done on the line above

Reply:
```
x, y: Tuple[x_type, y_type] = ... # invalid
x, y: x_type, y_type = ... # invalid
x: x_type, y: y_type = ... # invalid

x, y = func(YTypeConstructor) # type checker is not smart enough to know that `y` should have `y_type` despite the type variable hints.
```

```
tup : Tuple[x_type, y_type] = ...
x, y = tup # this typechecks
```
The above seems like a minimal concession, and I don't know of a better way to unpack while annotating types.

Reply:
OK nvm the typechecker is smarter than I thought